src/org/usfirst/frc/team2706/robot/RobotMap.java

```diff
@@ -160,6 +160,14 @@
     private static final int[] INFRARED_SENSOR_ANALOG_VALS = {1, 1, 1};
     public static final int INFRARED_SENSOR_ANALOG = getConstant("INFRARED_SENSOR_ANALOG");
     
+    // limit switch left
```

Comment:
To keep this consistent with the other comments, try to capitalize the first word in the comments